user/container/Makefile

```diff
@@ -57,18 +58,21 @@ run:
 	$(info Arguments: NAME=${NAME})
 	$(info Arguments: BASEPATH=${BASEPATH})
 	$(info Arguments: EXTERNAL_HOSTNAME=${EXTERNAL_HOSTNAME})
-	mkdir -p ${BASEPATH}/.kcp ${BASEPATH}/kubestellar-logs
+	$(info Arguments: EXTERNAL_PORT=${EXTERNAL_PORT})
 	docker run -d \
-		-p 6443:6443 \
-		-v ${BASEPATH}/.kcp:/.kcp \
-		-v ${BASEPATH}/kubestellar-logs:/kubestellar-logs \
+		-p ${EXTERNAL_PORT}:6443 \
 		-e EXTERNAL_HOSTNAME=${EXTERNAL_HOSTNAME} \
+		-e EXTERNAL_PORT=${EXTERNAL_PORT} \
 		--user $(shell id -u) \
-		--name ${NAME} ${IMG}:${TAG}
-	docker cp ${NAME}:/kcp-plugins ${BASEPATH}/kcp
-	docker cp ${NAME}:/kubestellar ${BASEPATH}/kubestellar
+		--name ${NAME} \
+		${IMG}:${TAG}
```

Comment:
There is a versioning inconsistency here.  This particular version of this Makefile has particular expectations of what the image does, but `${TAG}` causes the use of a potentially inconsistent image (controlled by the setting of "stable" on the website).

Reply:
@MikeSpreitzer I'm not understanding your comment, can you please clarify?

Reply:
The definition that I read above for `TAG` involves, when the user does not specify `KUBESTELLAR_VERSION`, going to the website whenever `make` is run to find the release that is marked as stable and using the container image from that release.  But this particular Makefile is written for particular behavior of the container image.  Unless we fix that behavior forever, there is a problem in the future here.  Consider a user who has fetched this Makefile and read the instructions and made a practice --- perhaps codified in their own automation --- of following this PR's instructions with this PR's Makefile.  One day in the future we change what is marked as "stable" on the website to some new and improved container image behavior.  That user then suffers breakage, using the old Makefile and instructions with the new container image.